src/main/java/com/northconcepts/datapipeline/foundations/examples/jdbc/JoinCsvFiles.java

```diff
@@ -0,0 +1,87 @@
+package com.northconcepts.datapipeline.foundations.examples.jdbc;
+
+import com.northconcepts.datapipeline.core.DataReader;
+import com.northconcepts.datapipeline.core.DataWriter;
+import com.northconcepts.datapipeline.core.StreamWriter;
+import com.northconcepts.datapipeline.csv.CSVReader;
+import com.northconcepts.datapipeline.csv.CSVWriter;
+import com.northconcepts.datapipeline.jdbc.JdbcConnectionFactory;
+import com.northconcepts.datapipeline.jdbc.JdbcReader;
+import com.northconcepts.datapipeline.jdbc.JdbcWriter;
+import com.northconcepts.datapipeline.jdbc.sql.select.Select;
+import com.northconcepts.datapipeline.job.Job;
+import com.northconcepts.datapipeline.transform.BasicFieldTransformer;
+import com.northconcepts.datapipeline.transform.TransformingReader;
+
+import java.io.File;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+
+public class JoinCsvFiles {
+
+    public static void main(String[] args) throws Throwable {
+
+        Connection connection = JdbcConnectionFactory.wrap("org.h2.Driver", "jdbc:h2:mem:jdbcTableSort;MODE=MySQL", "sa", "").createConnection();
```

Comment:
Can we use an on-disk database instead of an in-memory one?

Reply:
done